ibm_catalog.json

```diff
@@ -17,31 +17,31 @@
       "short_description": "Deploy watsonx.ai SaaS to automate the deployment and setup of the IBM watsonx platform.",
       "long_description": "The watsonX.ai SaaS Automation with Assistant and Governance easily deploys and embeds AI across your business, manages all data sources, and accelerates responsible AI workflows on one platform.",
       "provider_name": "FinOps Team.",
-      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-saas-da/blob/main/README.md",
+      "offering_docs_url": "https://test.cloud.ibm.com/docs-draft/watsonx-ai-saas-automation?topic=watsonx-ai-saas-automation-responsibilities",
```

Comment:
This link will have to be updated to https://cloud.ibm.com/docs-draft/watsonx-ai-saas-automation?topic=watsonx-ai-saas-automation-responsibilities when we push to production

Reply:
Why docs-draft? Shouldn't be `docs`?